README.md

```diff
@@ -1,2 +1,56 @@
 # m-agent
-LitmusChaos Machine Agent
+Machine Agent a.k.a. m-agent is a lightweight, platform-generic daemon agent that can remotely inject faults into Guest-OS scoped resources, as part of the Litmus Chaos Experiments.
```

Comment:
I will suggest machine scoped resources since we will be also injecting failure into a disk, network, etc. which although are managed by the os but can't be termed as OS resources.